.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: WebView2 issues
+    url: https://github.com/MicrosoftEdge/WebView2Feedback/issues
+    about: Use the WebView2Feedback repository to submit feature suggestions or bugs about the WebView2 control, or to search for known issues. The repository is a place for developers to share tips and best practices, report feedback, or search existing issues for WebView2.
+  - name: DevTools issues
+    url: https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/contact
+    about: To submit feature requests or bug reports about Microsoft Edge DevTools, please contact the DevTools team.
```

```suggestion
    about: To submit feature requests or bug reports about Microsoft Edge DevTools, contact the DevTools team.
```
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/please